packages/forklift-console-plugin/tsconfig.json

```diff
@@ -1,22 +1,25 @@
 {
   "extends": "../../config/tsconfig.base.json",
 
-  "include": ["src/**/*", "*.ts"],
+  "include": ["./src/**/*", "./plugin-metadata.ts", "./plugin-extensions.ts"],
 
   "compilerOptions": {
-    "outDir": "dist",
-    "declarationDir": "dist/types",
+    "outDir": "./dist",
 
     "paths": {
       "src/*": ["./src/*"],
 
-      "common/src/*": ["../common/src/*"],
-      "legacy/src/*": ["../legacy/src/*"]
+      "@kubev2v/legacy/*": ["../legacy/src/*"],
+      "@kubev2v/common/*": ["../common/src/*"],
+      "@kubev2v/mocks/*": ["../mocks/src/*"],
+      "@kubev2v/webpack": ["../webpack/src"],
+      "@kubev2v/types": ["../types/src"]
```

Comment:
This essentially overlays those packages to the plugin's source tree.  I think it negates the project references entirely.

:man_shrugging:

Reply:
yes, this should only happen in dev mode
unfortunately this also speed compile time from aprox 6m to aprox 1m
so I was not in a hurry to make it dev only

in production, we should use minified compiled packages and not do this hack